01week/resume/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<link rel="stylesheet" href="./css/style.css">
```

Comment:
I think you and I already talked about this, so sorry for the delayed note, but don't forget your <head> tag. Remember, the stuff that goes in this tag (like the <link> tag) is stuff the webpage needs to work, but it won't show up on the page when people view it.